[2025-09-06T20:33:17.932+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2025-09-06T20:33:15.969801+00:00 [queued]>
[2025-09-06T20:33:17.958+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2025-09-06T20:33:15.969801+00:00 [queued]>
[2025-09-06T20:33:17.960+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-09-06T20:33:18.011+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): stream_data_from_api> on 2025-09-06 20:33:15.969801+00:00
[2025-09-06T20:33:18.024+0000] {standard_task_runner.py:57} INFO - Started process 586 to run task
[2025-09-06T20:33:18.031+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'user_automation', 'stream_data_from_api', 'manual__2025-09-06T20:33:15.969801+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream.py', '--cfg-path', '/tmp/tmp3zy69d_t']
[2025-09-06T20:33:18.037+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask stream_data_from_api
[2025-09-06T20:33:18.265+0000] {task_command.py:416} INFO - Running <TaskInstance: user_automation.stream_data_from_api manual__2025-09-06T20:33:15.969801+00:00 [running]> on host 9a4fec9476f6
[2025-09-06T20:33:18.739+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='user_automation' AIRFLOW_CTX_TASK_ID='stream_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T20:33:15.969801+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T20:33:15.969801+00:00'
[2025-09-06T20:33:19.060+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: connecting to broker:29092 [('172.18.0.9', 29092) IPv4]
[2025-09-06T20:33:19.063+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2025-09-06T20:33:19.066+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: Connection complete.
[2025-09-06T20:33:19.185+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2025-09-06T20:33:19.194+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2025-09-06T20:33:20.078+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: connecting to broker:29092 [('172.18.0.9', 29092) IPv4]
[2025-09-06T20:33:20.094+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: Connection complete.
[2025-09-06T20:33:20.105+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connected> [IPv4 ('172.18.0.9', 29092)]>: Closing connection. 
[2025-09-06T20:33:40.638+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=broker:29092 <connected> [IPv4 ('172.18.0.9', 29092)]>: Closing connection. KafkaConnectionError: Socket EVENT_READ without in-flight-requests
[2025-09-06T20:33:40.649+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-06T20:33:41.158+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: connecting to broker:29092 [('172.18.0.9', 29092) IPv4]
[2025-09-06T20:33:41.167+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]> returned error 111. Disconnecting.
[2025-09-06T20:33:41.171+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-06T20:33:41.176+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-06T20:33:41.185+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: connecting to broker:29092 [('172.18.0.9', 29092) IPv4]
[2025-09-06T20:33:41.194+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]> returned error 111. Disconnecting.
[2025-09-06T20:33:41.201+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-06T20:33:41.204+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-06T20:33:41.419+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: connecting to broker:29092 [('172.18.0.9', 29092) IPv4]
[2025-09-06T20:33:41.423+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]> returned error 111. Disconnecting.
[2025-09-06T20:33:41.426+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.9', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-06T20:33:41.435+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-06T20:33:42.201+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:42.204+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:42.346+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:42.348+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:42.795+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:42.797+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:43.248+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:43.251+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:43.699+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:43.701+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:44.152+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:44.153+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:44.611+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:44.612+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:45.059+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:45.060+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:45.543+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:45.563+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:45.974+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:45.976+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:46.425+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:46.426+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:46.747+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:46.880+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:46.881+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:47.330+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:47.331+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:47.789+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:47.790+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:48.237+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:48.238+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:48.690+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:48.692+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:49.142+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:49.143+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:49.597+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:49.598+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:50.049+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:50.050+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:50.501+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:50.502+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:50.770+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:50.954+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:50.955+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:51.205+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:51.409+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:51.410+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:51.633+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:51.862+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:51.862+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:52.079+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:52.315+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:52.316+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:52.500+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:52.770+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:52.772+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:53.221+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:53.222+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:53.262+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:53.675+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:53.676+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:53.692+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:54.128+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:54.129+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:54.581+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:54.581+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:55.036+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:55.038+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:55.489+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:55.490+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:55.941+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:55.942+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:56.410+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:56.412+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:56.852+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:56.853+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:57.276+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:57.304+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:57.306+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:57.712+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:57.760+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:57.764+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:58.137+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:58.231+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:58.234+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:58.659+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:58.677+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:58.678+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:59.093+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:33:59.121+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:59.122+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:33:59.574+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:33:59.575+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:00.027+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:00.029+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:00.481+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:00.482+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:00.830+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:00.933+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:00.934+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:01.260+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:01.386+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:01.387+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:01.700+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:01.841+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:01.842+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:02.182+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:02.293+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:02.294+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:02.609+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:02.747+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:02.748+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:03.029+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:03.198+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:03.199+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:03.657+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:03.659+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:04.111+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:04.116+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:04.557+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:04.558+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:04.793+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:05.014+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:05.017+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:05.223+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:05.474+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:05.475+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:05.657+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:05.918+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:05.919+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:06.091+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:06.372+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:06.374+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:06.822+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:06.823+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:07.276+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:07.277+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:07.732+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:07.733+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:08.190+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:08.192+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:08.325+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:08.634+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:08.635+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:09.089+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:09.090+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:09.542+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:09.543+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:09.996+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:09.998+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:10.450+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:10.453+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:10.818+0000] {record_accumulator.py:73} WARNING - Produced messages to topic-partition ('users_created', 0) with base offset -1 log start offset None and error None.
[2025-09-06T20:34:10.818+0000] {record_accumulator.py:332} WARNING - Expired 1 batches in accumulator
[2025-09-06T20:34:10.899+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:10.900+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:11.288+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:11.352+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:11.353+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:11.717+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:11.806+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:11.807+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:12.260+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:12.261+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:12.711+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:12.712+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:13.164+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:13.165+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:13.618+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:13.619+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:14.071+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:14.072+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:14.525+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:14.526+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:14.783+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:14.977+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:14.979+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:15.248+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:15.440+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:15.446+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:15.885+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:15.886+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:16.345+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:16.346+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:16.795+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:16.796+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:17.245+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:17.246+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:17.697+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:17.698+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:18.051+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-09-06T20:34:18.054+0000] {process_utils.py:131} INFO - Sending 15 to group 586. PIDs of all processes in the group: [586]
[2025-09-06T20:34:18.055+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 586
[2025-09-06T20:34:18.056+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-09-06T20:34:18.057+0000] {kafka_stream.py:60} ERROR - An error occured: Task received SIGTERM signal
[2025-09-06T20:34:18.150+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:18.151+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:18.461+0000] {kafka_stream.py:60} ERROR - An error occured: list index out of range
[2025-09-06T20:34:18.606+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:18.607+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:19.057+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:19.058+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:19.511+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:19.512+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:19.965+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:19.966+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:20.417+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:20.418+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:20.869+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:20.870+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:21.323+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:21.323+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:21.776+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:21.777+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:22.229+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:22.230+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:22.683+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:22.683+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:23.134+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:23.135+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:23.589+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:23.590+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:24.042+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:24.043+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:24.494+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:24.495+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:24.957+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:24.961+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:25.406+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:25.407+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:25.885+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:25.886+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:26.308+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:26.308+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:26.761+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:26.762+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:27.213+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:27.214+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:27.675+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:27.677+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:28.152+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:28.153+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:28.651+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:28.652+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:29.161+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:29.162+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:29.570+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:29.571+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:29.931+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:29.932+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:30.386+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:30.387+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:30.840+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:30.841+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:31.292+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:31.293+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:31.745+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:31.746+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:32.197+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:32.198+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:32.650+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:32.652+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:33.103+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:33.104+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:33.558+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:33.559+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:34.010+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:34.011+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:34.463+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:34.464+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:34.916+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:34.917+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:35.371+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:35.372+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:35.823+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:35.825+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:36.278+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:36.279+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:36.751+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:36.752+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:37.251+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:37.252+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:37.646+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:37.647+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:38.141+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:38.142+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:38.549+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:38.550+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:39.063+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:39.064+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:39.453+0000] {conn.py:1527} WARNING - DNS lookup failed for broker:29092, exception was [Errno -2] Name or service not known. Is your advertised.listeners (called advertised.host.name before Kafka 9) correct and resolvable?
[2025-09-06T20:34:39.453+0000] {conn.py:315} ERROR - DNS lookup failed for broker:29092 (0)
[2025-09-06T20:34:39.610+0000] {record_accumulator.py:73} WARNING - Produced messages to topic-partition ('users_created', 0) with base offset -1 log start offset None and error None.
[2025-09-06T20:34:39.611+0000] {record_accumulator.py:332} WARNING - Expired 1 batches in accumulator
[2025-09-06T20:34:39.663+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-09-06T20:34:39.677+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_automation, task_id=stream_data_from_api, execution_date=20250906T203315, start_date=20250906T203317, end_date=20250906T203439
[2025-09-06T20:34:39.708+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=586, status='terminated', exitcode=0, started='20:33:17') (586) terminated with exit code 0
